app/helpers/welcome_helper.rb

```diff
@@ -1,2 +1,12 @@
 module WelcomeHelper
+  #getRank returns a colour in relation to the skill rank that has ben set
+  def getRank(sk)
```

Comment:
try to stay away from naming functions like "getThing" as it's not considered the 'railsy' way
instead just call it "thing"